game/definitions.rpy

```diff
@@ -751,8 +750,48 @@ init -999 python in utils:
         """
         return pprint.pformat(object.__dict__, indent, width)
 
+    def jn_get_mouse_position():
+        """
+        Returns a tuple representing the mouse's current position in the game window.
+
+        OUT:
+            - mouse position as a tuple in format (x,y)
+        """
+        return pygame.mouse.get_pos()
+
 init python in utils:
-    def get_current_session_length():
+    import easter
+    from Enum import Enum
+
+    TIME_BLOCK_EARLY_MORNING = 0
+    TIME_BLOCK_MID_MORNING = 1
+    TIME_BLOCK_LATE_MORNING = 2
+    TIME_BLOCK_AFTERNOON = 3
+    TIME_BLOCK_EVENING = 4
+    TIME_BLOCK_NIGHT = 5
+
+    _easter = easter.easter(datetime.datetime.today().year)
+
+    JN_NEW_YEARS_DAY = datetime.date(datetime.date.today().year, 1, 1)
+    JN_EASTER = datetime.date(_easter.year, _easter.month, _easter.day)
+    JN_HALLOWEEN = datetime.date(datetime.date.today().year, 10, 31)
+    JN_CHRISTMAS_EVE = datetime.date(datetime.date.today().year, 12, 24)
+    JN_CHRISTMAS_DAY = datetime.date(datetime.date.today().year, 12, 25)
+    JN_NEW_YEARS_EVE = datetime.date(datetime.date.today().year, 12, 31)
```

Comment:
We can keep these in the global store and likely just `define` them since they carry the global prefix